### ISSUE TITLE
Add “Create test submission” button submission portal environments

### ISSUE THREAD

**Issue by mslarae13** [microbiomedata/issues#827]
In replacement of the sandbox environment, we want to allow people to make test submissions to NMDC Submission portal environments. 
Deletion policy for test submissions are the same as real submissions (as in there is none. Never auto-delete)

Additionally, we need to track WHAT submissions are tests (test = true) for metrics.

Suggestion
![Screenshot 2024-08-15 at 10 44 41 AM](https://github.com/user-attachments/assets/e0966790-d7b7-4c0a-bb76-45dab5b76476)

"i" tooltip should be: 
"Test submissions: 
- are for workshops, examples, testing, or any submission that is **not** for a real study, metadata, and sample submission
- will be accessible at any time
- can **not** be submitted
- can **not** be converted to real submissions
- are **not** equal to Draft

**Comment by mslarae13** [comment]
I think we should also change "+ Create New Submission" button to just "+ Create Submission"

Anyone opposed?
@emileyfadrosh @lamccue @jkelliher-github @ljohnson09 @frodriguez16 @simroux

**Comment by mslarae13** [comment]
Any other information we need in the tooltip?
@emileyfadrosh @lamccue @jkelliher-github @ljohnson09 @frodriguez16 @simroux

**Comment by emileyfadrosh** [comment]
@mslarae13 I agree with changing "Create New Submission" to just "Create Submission" 

For the tooltip, is it the bullet point list above? I am confused by the "cannot be converted to a real submission" --> wouldn't that be a potential desired behavior? Also, confused by "will be accessible at any time" since I thought it would be removed after a week? Thanks for clarifying.

**Comment by lamccue** [comment]
@emileyfadrosh 
We are eliminating the weekly wipe of the test submissions. We decided that we want some time to observe user behavior and how this new approach to the portal works, since users will have the ability to delete their own entries.
Also, a user will have the ability to create a real submission in draft form, and if they want to, delete it before they hit submit. This keeps the test submissions as true tests and a playground for demonstrations. Anyone who is serious and wants to start a draft submission should do it in the real submissions.

**Comment by simroux** [comment]
For "not for use to draft real metadata submissions", maybe we can clarify "not for use to draft real metadata submissions (use 'Create Submission' instead for these)".
Everything else sounds good, and the tooltip should (hopefully) be clear enough

**Comment by ljohnson09** [comment]
Very exciting! +1 to Lee Ann's suggestion for tool tip w/ one tiny comment: 
 
> * entries will not be submitted to the NMDC metadata store

Would the DH interface for a test submission not have a functional submit button? Or if someone clicks submit on the test submission, @mslarae13 deletes any test = true submissions later?

**Comment by mslarae13** [comment]
>Would the DH interface for a test submission not have a functional submit button? 

Correct. Test does not equal draft.
You cannot submit a test submission

**Comment by JamesTessmer** [comment]
Associated PR here: https://github.com/microbiomedata/nmdc-server/pull/1495

**Comment by mslarae13** [comment]
@JamesTessmer think this will be ready for testing and complete in time for January release?

**Comment by JamesTessmer** [comment]
@mslarae13 Just merged in, changes should be on dev.

**Comment by mslarae13** [comment]
Yeah, it auto marks and complete and I change it to "pending ingest"
